index.md

```diff
@@ -2,24 +2,23 @@
 
 Welcome to Helix Pages!
 
-To use it, change the current URL to `https://<repo>--<owner>.project-helix.page`.
-`<owner>` and `<repo>` must refer to a valid Git repository.
+No sign-up: All you need is a public GitHub repository, then you can navigate to `https://<ref>--<repo>--<owner>.hlx.page`
+(`<owner>`, `<repo>` and `<ref>` must refer to a valid GitHub repository).
 
-Example: <https://helix-home--adobe.project-helix.page/README.html>
+Example: <https://main--helix-home--adobe.hlx.page/>
```

Comment:
I would keep the `README.html` since `index` does not exist.

Reply:
Yes, it does: https://github.com/adobe/helix-home/blob/main/index.html